pandas/core/groupby/ops.py

```diff
@@ -783,6 +783,30 @@ def indices(self) -> dict[Hashable, npt.NDArray[np.intp]]:
         keys = [ping.group_index for ping in self.groupings]
         return get_indexer_dict(codes_list, keys)
 
+    @final
+    def result_ilocs(self) -> npt.NDArray[np.intp]:
+        """
+        Get the original integer locations of result_index in the input.
+        """
+        # Original indices are where group_index would go via sorting.
+        # But when dropna is true, we need to remove null values whilst accounting for
```

Comment:
"whilst" may be tough for non-native speakers

Reply:
Thanks - changed to while